code/modules/modular_computers/laptop_vendor.dm

```diff
@@ -92,6 +96,19 @@
 				if(fabricate)
 					fabricated_laptop.install_component(new /obj/item/weapon/computer_hardware/network_card/advanced)
 				total_price += 299
+		switch(dev_portable)
+			if(1) // Basic(16QC)
```

Comment:
I'd much rather these be defines than magic numbers... but thats the case for all of these vars :(